Source/Shaders/ModelExperimental/SkinningStageVS.glsl

```diff
@@ -0,0 +1,18 @@
+void skinningStage(inout ProcessedAttributes attributes) 
+{
+    mat4 skinningMatrix = getSkinningMatrix();
+    mat3 skinningMatrixMat3 = mat3(skinningMatrix);
+
+    vec4 positionMC = vec4(attributes.positionMC, 1.0);
+    attributes.positionMC = vec3(skinningMatrix * positionMC);
+
+    #ifdef HAS_NORMALS
+    vec3 normalMC = attributes.normalMC;
+    attributes.normalMC = skinningMatrixMat3 * normalMC;
```

Comment:
Just to check my understanding, this works because the normal matrix is applied later in the Geometry stage, right?

Reply:
Yes